pandas/tests/groupby/test_function.py

```diff
@@ -685,59 +685,53 @@ def test_numpy_compat(func):
         getattr(g, func)(foo=1)
 
 
-def test_cummin_cummax():
+@pytest.mark.parametrize("dtype", [np.int32, np.int64, np.float32, np.float64])
```

Comment:
we have fixture for these

Reply:
Which fixture could I use?

Reply:
I _think_ you could try any_real_dtype and see if the tests pass

Reply:
Looks like it breaks for the unsigned real dtypes

Reply:
OK, not sure we want to create another fixture, add a skip in the test for the unsigned or keep the dtype as direct parameterisation and include the min and max in the parameterisation. @jreback?